test/verifiedTests/dataIntegration/testEFlux/Aerobic.txt

```diff
@@ -0,0 +1,4298 @@
+^SAMPLE = GSM1126445

```

Comment:
May you fetch this data from somewhere else? What is the source of this test data?

Reply:
I'll update it to download the data from GEO